### PR TITLE
added tachyons class

### PR DESCRIPTION
**Pull request by Ienzi**
#### How to test it? \* This is the workspace: https://checkoutiopr--ukccwhirlpool.myvtex.com/cart#/ . To replicate: add a product and go to the cart. this is without the className: 
image
 and this with the class added: 
image

#### Describe alternatives you've considered, if any. \* 
- Considered other names for such class;
- Tried with cssHandles but failed because of a dependency cycle detected: 
image

### CODE REVIEW

**Comment by danzanzini** [comment]
Hello @Ienzi, can you clarify what color this new class is changing? I couldn't see by the images you sent

File: react/components/ExtensionPoint/index.tsx [hunk 1]
```diff
@@ -161,7 +161,7 @@ function withOuterExtensions(
     <Fragment key={`wrapped-${treePath}`}>
       <LazyImages>{beforeElements}</LazyImages>
       {element}
-      {isRootTreePath && <div className="flex flex-grow-1" />}
+      {isRootTreePath && <div className="flex flex-grow-1 bg-checkout-color" />}
```
Comment (danzanzini) [inline react/components/ExtensionPoint/index.tsx:164]: An ExtensionPoint is used in many situations, so the name should be more generic than this.

```suggestion
      {isRootTreePath && <div className="flex flex-grow-1 below-element" />}
```

**Review by danzanzini — CHANGES_REQUESTED** [review]
Hey @Ienzi, thanks for the contribution.

Left one suggestion for the name of the class.

**Comment by Ienzi** [comment]
Hello @danzanzini, I've confirmed your changes, please keep me posted!

**Review by danzanzini — APPROVED** [review]

**Comment by danzanzini** [comment]
Hello @Ienzi. Just did the merge. Can you install it on your store and check if everything is ok?

**Comment by Ienzi** [comment]
Hi @danzanzini in order to work it should be "bg-below-element" not just "below-element"

**Pull request merged**